controller/test/service/api/actuator/test_basic_actuator.py

```diff
@@ -61,15 +61,22 @@ def locator_instance_does_not_exist(self, vm_id):
         else:
             raise InstanceNotFoundException(vm_id)
 
+    @unittest.skip("Authorization isn't working")
+    def test_authorization(self):
+        self.actuator.authorizer.get_authorization = MagicMock(
+            return_value={'success': True})
+        
+        # Actuator tries to authenticate
+        self.actuator.authorizer.get_authorization.assert_called_once_with(self.authorization_url,
+                                                                           self.bigsea_username, self.bigsea_password)
+
     def test_adjust_resources_locates_and_acts_correctly_1_instance(self):
         vm_data = {self.vm_id1: self.cap1}
 
         self.instance_locator.locate = MagicMock(return_value=self.host_ip1)
         self.remote_kvm.change_vcpu_quota = MagicMock(return_value=None)
         self.remote_kvm.change_io_quota = MagicMock(return_value=None)
-        self.actuator.authorizer.get_authorization = MagicMock(
```

Comment:
I see some tests were skipped due to the lack of a working implementation of the authentication feature, and that is ok for now. I'm just thinking that in other test methods that use the authentication process we could comment out this info rather than deleting it altogether, since I feel we might use these lines again in the future once we have authentication. Is there any other reason why you deleted these lines?  I see a couple of lines like this in the rest of the file as well..